pkg/common/common.go

```diff
@@ -16,26 +16,29 @@ package common
 
 import (
 	"strings"
+
+	corev1 "k8s.io/api/core/v1"
 )
 
 const (
```

Comment:
also make sure that legacy webhook annotations (the previous version) are also supported and used as a fallback option in case the new one is not specified to allow backwards compatibility